### PR TITLE
Fix some shellcheck linter issues in shell scripts

### PR DESCRIPTION
**Pull request by anewmanRH**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 443b119e6f315050b32ae975ef7e0a11a99be191:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by anewmanRH** [comment]
More info on shellcheck can be found here.
https://github.com/koalaman/shellcheck

File: sprout/sprout.sh [hunk 1]
```diff
@@ -368,7 +369,7 @@ function start_flower() {
     cddir
     if [ -e "${PIDFILE_FLOWER}" ] ;
     then
-        ps -p `cat $PIDFILE_FLOWER` >/dev/null
+        ps -p "cat $PIDFILE_FLOWER" >/dev/null
```
Comment (mfalesni) [inline sprout/sprout.sh]: This looks slightly weird to me, since inside `""` it won't execute the `cat` ... or yes?

**Comment by dajoRH** [comment]
Lint report for commit 256f524a194cdd8421e6726342d0752bae793193:

Everything seems all right :smile: .
*CFME QE Bot*

**Comment by psav** [comment]
@mfalesni looks like you reviewed it, green to go?

**Comment by mfalesni** [comment]
I think this is good to go.

**Pull request merged**